subprojects/snapshots/src/main/java/org/gradle/internal/snapshot/PathUtil.java

```diff
@@ -167,13 +172,13 @@ static boolean equalChars(char char1, char char2, CaseSensitivity caseSensitivit
      *
      * The size of the common prefix does not include the last line separator.
      */
```

Comment:
It would be great to add some examples here, too.

Reply:
I added some samples.